starboard/android/shared/player_components_factory.h

```diff
@@ -235,9 +226,9 @@ class PlayerComponentsFactory : public starboard::shared::starboard::player::
     SB_LOG(INFO) << "Creating passthrough components.";
     // TODO: Enable tunnel mode for passthrough
     scoped_ptr<AudioRendererPassthrough> audio_renderer;
-    audio_renderer.reset(new AudioRendererPassthrough(
-        creation_parameters.audio_stream_info(),
-        creation_parameters.drm_system(), enable_audio_device_callback));
+    audio_renderer.reset(
+        new AudioRendererPassthrough(creation_parameters.audio_stream_info(),
+                                     creation_parameters.drm_system(), true));
```

Comment:
You can remove this param from AudioRenderPassthrough as well

Reply:
Fixed it.